packaging_uom/views/product_packaging_views.xml

```diff
@@ -11,7 +11,7 @@
                 <page name="packaging" string="Packaging"  attrs="{'invisible':[('type','=','service')]}" groups="product.group_stock_packaging" >
                     <group name="packaging" string="Packaging">
                         <field name="packaging_ids" string="Configurations"
-                               context="{'get_uom_categ_from_uom': parent.uom_id, 'tree_view_ref':'product.product_packaging_tree_view_product', 'form_view_ref': 'product.product_packaging_form_view_without_product'}"/>
+                               context="{'tree_view_ref':'product.product_packaging_tree_view_product', 'form_view_ref': 'product.product_packaging_form_view_without_product'}"/>
```

Comment:
@jhumfer Ok you deleted it, but I think 'default_product_id' is not defined when creating packaging in existing product form, so the when does the domain function '_default_uom_categ_domain_id' work ?

Reply:
Have you tested it in the interface?

Reply:
Yes, I have tested it in the interface and it works correctly. This ensures that 'default_product_id' is defined when creating the package in the existing product form

Reply:
Just tested in the interface.

Ok through product form.

KO through the product packages menu (create from scratch). The domain should be correctly computed after having selected the product.

Reply:
Fixed from product packages menu

Reply:
Is there a problem for the validation?